Makefile

```diff
@@ -40,16 +40,12 @@ ifneq ($(T1),)
 $(T1):
```

Comment:
as a part of generic Makeflies cleanup:
 - You can remove all these `TARGET ?= ` a the beginning of the file
 - `$(T1)` rule can be simplified: https://github.com/phoenix-rtos/phoenix-rtos-devices/commit/2bb99186b080d99f5b8dfd6571d0a4d499e96846#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52